docs/computing/connecting/index.md

```diff
@@ -0,0 +1,103 @@
+# Connecting to CSC supercomputers
+
+There are two main ways of connecting to CSC supercomputers. The traditional way to
+work on a supercomputer is [using an SSH client](#using-an-ssh-client). We
+also offer a [web interface](#using-the-web-interface) to our systems, which
+enables running both graphical applications and command-line shells.
+
+In addition to these methods, it is also possible to use certain source code
+editors, such as Visual Studio Code, to
+[develop scripts remotely](../../support/tutorials/remote-dev.md).
```

Comment:
Do we want to advertise this? It is possible indeed, but it is unclear to me whether this is better than using VSCode / Jupyter via web interfaces.

Reply:
I would think so, since we have a tutorial for it :thinking:

Reply:
Removed in ca30c8e5

Reply:
Added back in 30a0837f